src/coreComponents/physicsSolvers/contact/SolidMechanicsALMKernels.hpp

```diff
@@ -223,17 +226,13 @@ class ALM :
       }
     }
 
-    // The minus sign is consistent with the sign of the Jacobian
-    stack.localPenalty[0][0] = -m_penalty( k, 0 );
-    stack.localPenalty[1][1] = -m_penalty( k, 1 );
-    stack.localPenalty[2][2] = -m_penalty( k, 1 );
-
     for( int i=0; i<numTdofs; ++i )
     {
       stack.tLocal[i] = m_traction( k, i );
       stack.dispJumpLocal[i] = m_dispJump( k, i );
       stack.oldDispJumpLocal[i] = m_oldDispJump( k, i );
     }
+    stack.oldDispJumpLocal[0] = 0.0;
```

Comment:
a comment here would not hurt

Reply:
You're right. This was a remnant of the initial implementation. I originally did this because the old displacements were only used to update the tangential traction. Since the normal traction doesn't depend on the previous normal displacement, I set it to 0. However, the current implementation updates the traction inside a dedicated function, so this line is no longer necessary. I'll remove it.